python/ray/llm/tests/batch/stages/BUILD

```diff
@@ -1,8 +1,15 @@
 load("//bazel:python.bzl", "py_test_module_list")
 
 py_test_module_list(
-  files = glob(["test_*.py"]),
+  files = glob(["test_*.py"], exclude=["test_vllm_engine_stage.py"]),
```

Comment:
add comment on why this is excluded?

Reply:
do you expect people keep adding exclusions or is there a better way? 

maybe you should separate the tests folder structure to core and gpu??

Reply:
I just put it here but don't know what's the best way. Waiting for Lonnie to resolve the vLLM dependencies first.